src/utils/parseJsDoc.js

```diff
@@ -28,7 +29,14 @@ function getType(tag) {
   if (!tag.type) {
     return null;
   }
-  return {name: tag.type.name};
+  return {name: tag.type.name ? tag.type.name : tag.type.expression.name};
+}
+
+function getOptional(tag) {
+  if (tag.type && tag.type.type && tag.type.type === 'OptionalType') {
+    return true;
+  }
+  return;
```

Comment:
`option: getOptional(tag)` will add the property anyway. I mean, the property will disappear when converted to JSON, but it will still be on the JS object.